client/data/support-article-alternates/use-support-article-alternates-query.js

```diff
@@ -0,0 +1,29 @@
+import { isDefaultLocale } from '@automattic/i18n-utils';
+import { useQuery } from 'react-query';
+import { useSelector } from 'react-redux';
+import wp from 'calypso/lib/wp';
+import { isPostKeyLike } from 'calypso/reader/post-key';
+import getCurrentLocaleSlug from 'calypso/state/selectors/get-current-locale-slug';
+
+function useSupportArticleAlternatesQuery( blogId, postId, queryOptions = {} ) {
```

Comment:
This hook has only one usage, and it's a very specialized thing. I would consider inlining it into the support article dialog component.

Reply:
I don't see any downsides of keeping it separate, together with the rest of the custom queries. I prefer keeping it where it is if you don't mind.